README.md

```diff
@@ -57,13 +57,18 @@ yarn run coverage
 
 ## Releasing packages
 
-The release process is automated via CircleCI. Before you start, you need to prepare the changelog entries.
+The release process is automated via CircleCI and it can release both channels: stable (`X.Y.Z`) and pre-releases (`X.Y.Z-alpha.X`, etc.).
+
+Before you start, you need to prepare the changelog entries.
 
 1. Make sure the `#master` branch is up-to-date: `git fetch && git checkout master && git pull`.
 1. Prepare a release branch: `git checkout -b release-[YYYY-MM-DD]` where `YYYY-MM-DD` is the current day.
-1. Generate the changelog entries: `yarn run changelog --branch release-[YYYY-MM-DD]`.
-    * This task checks what changed in each package and bumps the version accordingly. If nothing changes at all, it won't create a new changelog entry. If changes were irrelevant (e.g., only dependencies), it would make an "_internal changes_" entry.
-    * Scan the logs printed by the tool to search for errors (incorrect changelog entries). Incorrect entries (e.g., ones without the type) should be addressed. You may need to create entries for them manually. This is done directly in CHANGELOG.md (in the root directory). Make sure to verify the proposed version after you modify the changelog.
+1. Generate the changelog entries: `yarn run changelog --branch release-[YYYY-MM-DD] [--from [GIT_TAG]]`.
+   * By default, the changelog generator uses the latest published tag as a starting point for collecting commits to process.
+     The `--from` modifier option allows overriding the default behavior. It is required when preparing the changelog entries for a next stable release while the previous one was marked as a prerelease, e.g., `@alpha`.
+     **Example**: let's assume that the `v40.5.0-alpha.0` tag is our latest and want to release it on a stable channel. The `--form` modifier should be equal to `--from v40.4.0`.
+   * This task checks what changed in each package and bumps the version accordingly. If nothing changes at all, it won't create a new changelog entry. If changes were irrelevant (e.g., only dependencies), it would make an "_internal changes_" entry.
+   * Scan the logs printed by the tool to search for errors (incorrect changelog entries). Incorrect entries (e.g., ones without the type) should be addressed. You may need to create entries for them manually. This is done directly in CHANGELOG.md (in the root directory). Make sure to verify the proposed version after you modify the changelog.
```

Comment:
```suggestion
1. Generate the changelog entries: `yarn run changelog --branch release-[YYYY-MM-DD] [--from [GIT_TAG]]`.
   * By default, the changelog generator uses the latest published tag as a starting point for collecting commits to process.
     The `--from` modifier option allows overriding the default behavior. It is required when preparing the changelog entries for the next stable release while the previous one was marked as a prerelease, e.g., `@alpha`.
     **Example**: Let's assume that the `v40.5.0-alpha.0` tag is our latest and that we want to release it on a stable channel. The `--from` modifier should be equal to `--from v40.4.0`.
   * This task checks what changed in each package and bumps the version accordingly. It won't create a new changelog entry if nothing changes at all. If changes were irrelevant (e.g., only dependencies), it would make an "_internal changes_" entry.
   * Scan the logs printed by the tool to search for errors (incorrect changelog entries). Incorrect entries (e.g., ones without the type) should be addressed. You may need to create entries for them manually. This is done directly in CHANGELOG.md (in the root directory). Make sure to verify the proposed version after you modify the changelog.
```